tools/doc-generator/doclet/src/main/java/org/projectnessie/nessie/docgen/SmallRyeConfigPropertyInfo.java

```diff
@@ -144,10 +155,27 @@ public static String simplifiedTypeName(Property property) {
       if (rawType == Character.class) {
         return "char";
       }
+      if (rawType == String.class) {
+        return "string";
+      }
+      if (rawType == Duration.class) {
+        return "duration";
+      }
+      if (rawType == Instant.class) {
+        return "instant";
+      }
+      if (rawType == URI.class) {
+        return "uri";
+      }
+      if (rawType == Path.class) {
+        return "path";
+      }
       return rawType.getSimpleName();
     }
     if (property.isGroup()) {
-      return property.asGroup().getGroupType().getInterfaceType().getSimpleName();
+      // Represents a type that consists of multiple fields/properties, which are documented
+      // underneath this property.
+      return "-";
```

Comment:
The single `-` char looks a bit obscure? Would something like `<composite>` work?

Reply:
I'd wanted to get rid of any of that - replaced with an empty string.